plugin-maven/src/test/java/com/diffplug/spotless/maven/java/RemoveWildcardImportsStepTest.java

```diff
@@ -0,0 +1,18 @@
+package com.diffplug.spotless.maven.java;
+
+import org.junit.jupiter.api.Test;
+
+import com.diffplug.spotless.maven.MavenIntegrationHarness;
+
+class RemoveWildcardImportsStepTest extends MavenIntegrationHarness {
+
+	@Test
+	void testRemoveWildcardImports() throws Exception {
+		writePomWithJavaSteps("<removeWildcardImports/>");
+
+		String path = "src/main/java/test.java";
+		setFile(path).toResource("java/removewildcardimports/JavaCodeWildcardsUnformatted.test");
+		mavenRunner().withArguments("spotless:apply").runNoError();
```

Comment:
```suggestion
		mavenRunner().withArguments("spotless:check").runNoError();
```
could test `check` as well.

Reply:
Thats nice suggestion but right now none of the other step-tests call `spotless:check`. so adding it here would make this one test an outlier

Reply:
> outlier

thanks for the learning.